components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -106,6 +108,7 @@ - (void)prepareForReuse {
 
   // Reset cells hidden during swipe deletion.
   self.hidden = NO;
+  [self.inkView cancelAllAnimationsAnimated:NO];
```

Comment:
Consider adding a line between this and self.hidden above to indicate that this is not part of
// Reset cells hidden during swipe detection
Or possibly move it into the earlier block with drawSparatorIfNeeded and updateInterfaceForEditing methods.